beetsplug/mbsubmit.py

```diff
@@ -35,21 +36,26 @@ class MBSubmitPlugin(BeetsPlugin):
     def __init__(self):
         super(MBSubmitPlugin, self).__init__()
 
+        self.config.add({
+            'mb_format': '$track. $title - $artist ($length)',
+            'threshold': 'medium',
+        })
+
+        # validate and store threshold
```

Comment:
We tend to format comments as "full sentences" in beets—starting with a capital letter and ending with a period.